src/autopas/utils/RaplMeter.cpp

```diff
@@ -19,7 +19,7 @@
 
 namespace autopas::utils {
 
-int RaplMeter::open_perf_event(int type, int config, int cpu) {
+int RaplMeter::open_perf_event(int type, int config, int cpu, std::string &errMsg) {
```

Comment:
done in [ffd9710](https://github.com/AutoPas/AutoPas/pull/786/commits/ffd971013de2f7bc75bd98e86a172d3d7f3382d2).
Only `RaplMeter::init()` returns an error string if something goes wrong. Additionally I added a check for permission in `RaplMeter::init()`, otherwise `open_perf_event` might throw during `RaplMeter::reset()`